src/SetHo.hs

```diff
@@ -127,6 +128,13 @@ runSetter mconfig rootName initialValue userPollForNewMessage sendRequest userCo
         makeStatsMessage >>= Gtk.labelSetText statsLabel
         userCommit counter val
 
+  let revertToDefaults val = do
```

Comment:
This is only used in one place, can you put it next to that place?

Reply:
Done.